inc/key_codes.h

```diff
@@ -0,0 +1,64 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   key_codes.h                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/12/19 22:45:45 by gwolf             #+#    #+#             */
+/*   Updated: 2023/12/19 22:47:15 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef KEY_CODES_H
+# define KEY_CODES_H
+
+// keyboard
+# define K_ENTER 65293
```

Comment:
The numbers are randomly chosen/predefined by minilibx?

Reply:
Kinda: The numbers are set by the X11 library, around which minilibx wraps. 
65293 may look nicer in hex: 0xff1b. It's only so high because it's a "special key". For example C has 0x63, which is 99, which is also it's value on the ASCII table.

I found out, that in context of X11 there is a difference between keycodes and keysymbols. 
- Keycode represents the physical hardware location: the first key, on the top row
- Keysymbol represents the higher layer: which function has a key, e.g. print C.
I'll rename the file.
The keys and symbols are not standardized. The "real" way of determining the key values and symbols would be to use a tool like xev or in programming the function XLookupKeysym(). Both we cannot do. But on the school machines as well as my WSL-Ubuntu, the keymappings are correct.
There is also X11/keysemdef.h which has the same Macros defined, but if I include it it doesn't find the macros! Maybe you get it to work?

Reply:
We can certainly try. Can bring my notebook tmr to campus

Reply:
Nevermind, I just forgot to include the Xlib.h header. Now we can use the definitions from keysymdef.h directly :D